gatsby-config.js

```diff
@@ -17,20 +16,6 @@ const {
 const isProduction = ENV === 'production'
 const siteUrl = isProduction ? URL : DEPLOY_PRIME_URL
 
-const unique = (x) => Array.from(new Set(x))
-
-const getSizes = (variants) =>
-  unique(
-    Object.values(variants)
-      .flatMap((variant) =>
-        variant.breakpoints.map((width) => [
-          `${width}x${Math.ceil(width / variant.aspectRatio)}`,
-          `${width}x${Math.floor(width / variant.aspectRatio)}`,
-        ])
-      )
-      .flat()
-  )
```

Comment:
Out of curiosity: what does this `getSizes` method was doing before, exactly? How/where/by whom these values were used?

Reply:
It was used by the proxy to avoid DDOS attacks. Now we are prone to this attack because there is no proxy in between the browser and the image server to avoid people resizing images